docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -116,6 +116,14 @@ to optimize how Determined interacts with PBS:
    other configurations such as targeting a specific resource pool with only GPU nodes, or
    specifying a PBS constraint in the experiment configuration.
 
+   PBS should be configured to provide the environment variable ``CUDA_VISIBLE_DEVICES``
+   (``ROCR_VISIBLE_DEVICES`` for ROCm) to the job based upon the GPUs allocated. If PBS is not
+   configured to provide ``CUDA_VISIBLE_DEVICES``, Determined will utilize a single GPU on each
+   node. To fully utilize mutliple GPUs, you must either manually define ``CUDA_VISIBLE_DEVICES`` or
```

Comment:
Do we need to say "``CUDA_VISIBLE_DEVICES`` environment variable" to make it clear that it's not a particular setting?

Reply:
Since it says 'environment variable' in the very first setting, it should be pretty clear.   Repeating it at each instance seemed to be too wordy.